apps/wallet-dashboard/app/(protected)/(feature-controlled)/vesting/page.tsx

```diff
@@ -342,10 +365,19 @@ export default function VestingDashboardPage(): JSX.Element {
                                     <Button
                                         type={ButtonType.Primary}
                                         text="Stake"
-                                        disabled={
-                                            supplyIncreaseVestingSchedule.availableStaking === 0n
-                                        }
                                         onClick={() => {
+                                            if (
+                                                supplyIncreaseVestingSchedule.availableStaking ===
+                                                    0n ||
+                                                new BigNumber(formattedAvailableStaking).lt(
+                                                    MIN_NUMBER_IOTA_TO_STAKE,
+                                                )
+                                            ) {
+                                                toast.error(
+                                                    'Not enough amount available for staking',
```

Comment:
nit: we could improve this message by rephrasing it as “Not enough funds available for staking” or “Insufficient amount available for staking.”
or NO_BALANCE_GENERIC_MESSAGE